src/components/CohortBuilder/ParticipantsTableView/ParticipantsTable.js

```diff
@@ -163,11 +181,24 @@ class ParticipantsTable extends Component {
     } = this.props;
     const { columns } = this.state;
     const selectedRowsCount = allRowsSelected ? dataTotalCount : selectedRows.length;
+    const projectId = arrangerProjectId;
 
     const handleRemoveFromCohort = () => {
       onRemoveFromCohort();
     };
 
+    const selectionSQON = this.props.selectedRows.length
```

Comment:
You can safely remove `replaceSQON` here since it does nothing.

```
export const replaceSQON: TMergeSQON = (q, ctxq) => {
  if (!ctxq && !q) return null;
  if (!ctxq) return q;
  if (!q) return ctxq;
```